### ISSUE TITLE
Make explicit import from tns-core-modules

### ISSUE THREAD

**Issue by jakobe** [shripalsoni04/nativescript-webview-interface#43]
In `index.d.ts` the current import is:
`import {WebView} from 'ui/web-view';`
This creates an implicit dependency in the project consuming the package where `tsconfig` need to have the following wildcard paths specified:
```
"compilerOptions": {
        "paths": {
            "*": [
                "./node_modules/tns-core-modules/*",
                "./node_modules/*"
            ],
        }
    },
````

This can easily be fixed by importing explicitly from `tns-core-modules`:
`import {WebView} from 'tns-core-modules/ui/web-view';`

**Comment by MortenGregersen** [comment]
Fixed by #44

**Comment by kdhrubo** [comment]
@shripalsoni04 pls merge and give a new minor release its breaking after upgrade

**Comment by shripalsoni04** [comment]
Thanks a lot @jakobe for the PR and @kdhrubo for a reminder.

I have merged the PR and it is released with version 1.4.3

**Comment by jakobe** [comment]
You're welcome  @shripalsoni04 and thanks for accepting the PR :)
